ffi_internals/src/items/impl_ffi.rs

```diff
@@ -33,14 +34,22 @@ pub struct ImplInputs {
     ///
     pub raw_types: Vec<Ident>,
 
-    /// The name of the trait that's implemented.
+    /// A hashmap whose keys are `Type`s for the generics used throughout this impl and whose
+    /// values are `Type`s for the concrete types to use in place of the generic for FFI.
     ///
-    /// Note that this is currently required; we don't support standalone impls right now because
-    /// we're relying on the trait name + type name pair to guarantee uniqueness for the generated
-    /// FFI module and consumer file. If we have a use case for exposing standalone impls, we'll
-    /// have to come up with another way to ensure that uniqueness.
+    /// # Limitations
     ///
-    pub trait_name: Ident,
+    /// This breaks down if an impl contains multiple generic functions that use the same generic
+    /// parameter but need to be matched with different concrete types. If we run into that use
+    /// case, we'll need to do something else (or perhaps let the function attributes override in
+    /// those cases?).
+    ///
```

Comment:
idk, do you think it's worth preemptively putting in tickets for things like this? Might be easier to have the marinate as an issue than a comment?